src/subscriptions/TyranoJumpProvider.ts

```diff
@@ -56,7 +56,11 @@ export class TyranoJumpProvider {
       document.lineAt(position.line).text,
     );
 
-    const TAGS_TO_JUMP = [
+    // TyranoScript syntax.tag.parameterから、{"tagName":"Path"}の形のObjectを作成
+    const tags: Object = await vscode.workspace
+      .getConfiguration()
+      .get("TyranoScript syntax.tag.parameter")!;
+    const enableJumpTags = [
```

Comment:
おそらく `merge` での選び間違いです（参照： #204 ）

```suggestion
    const TAGS_TO_JUMP = [ 
```

Reply:
ご指摘ありがとうございます。 🙇 
すみません、おっしゃる通り選び間違いでした 🙏